### DISCUSSION TITLE
export output static folder with nginx 403 Forbidden

### DISCUSSION THREAD

**Discussion by rain-sc** [amannn/next-intl#457] [Q&A]
### Description

I want to export static put in nginx to run.
I add a file `next.config.js`
```
/** @type {import('next').NextConfig} */
const nextConfig = {
  "output": "export",
}

module.exports = nextConfig
```
and to `npm run build` get a out folder
![螢幕截圖 2023-08-15 下午5 55 14](https://github.com/amannn/next-intl/assets/129497217/48b5d0df-1c8e-43c7-9886-1103f6619bc3)

after that the folder put in `/www` folder  for nginx
there is a config:
```
worker_processes  1;

events {
    worker_connections  1024;
}

http {
    include       mime.types;
    default_type  application/octet-stream;
    sendfile        on;
    keepalive_timeout  65;

    gzip on;
    gzip_types      text/plain application/xml;
    gzip_proxied    no-cache no-store private expired auth;
    gzip_min_length 1000;

    server {
        listen       3000;
        
        location / {
            root   html;
            index  index.html index.htm;
            expires 1d;
        }  
    }

    include servers/*;
}
```

when i press link : http://xxx.xxx.xxx.xxx:3000, it will redirect http://xxx.xxx.xxx.xxx:3000/en/  and error message is 403 Forbidden

### Mandatory reproduction URL (CodeSandbox or GitHub repository)

https://github.com/amannn/next-intl/tree/main/examples/example-next-13

### Reproduction description

Steps to reproduce:
1. Open reproduction
2. Click on …
3. See error: … 

### Expected behaviour

If successfully run in nginx, may be like: http://xxx.xxx.xxx.xxx:3000 or  http://xxx.xxx.xxx.xxx:3000/de
example:
https://next-intl-example-next-13.vercel.app/
https://next-intl-example-next-13.vercel.app/de

**Comment by amannn** [comment]
I'm unfortunately not an nginx expert myself. I'll convert this to a discussion since it's more a usage question.
